src/main/java/se/kth/sda/skeleton/post/Post.java

```diff
@@ -1,5 +1,46 @@
 package se.kth.sda.skeleton.post;
 
+import se.kth.sda.skeleton.user.User;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
 public class Post {
-    private int field;
```

Comment:
Why do we have a private field here?

Reply:
That was just to make a change to practice pushing from git. It's redundant and removed later.

Reply:
ok cool we can then just remove this later, but for now I will take the merge.